### DISCUSSION TITLE
Governance: approach to AIPs accepted in Aptos

### DISCUSSION THREAD

**Discussion by mzabaluev** [movementlabsxyz/movement#476] [Ideas]
(not sure which stage this belongs to yet)

We need to work out a way to keep up with Aptos Improvement Proposals that are accepted by Aptos governance.
Ideally, AIPs should be promptly converted to MIPs. Accepting the upstream proposals should normally be in the interest of the stakeholders, in order to keep the delta in aptos-core manageable.

**Comment by l-monninger** [comment]
This is a good one potential modify the template for in https://github.com/movementlabsxyz/MIP. I would not overspecify this here.

We actually have a first case of this with OKX requesting AIP-62.

**Reply by l-monninger** [reply]
@mzabaluev what I would do is submit an MD or MIP to include a description of a process for accepting upstream proposals and similar. It doesn't have to be lengthy or intricate.
